python/cugraph/cugraph/experimental/compat/nx/algorithms/link_analysis/pagerank_alg.py

```diff
@@ -0,0 +1,127 @@
+        weight="weight",
+        dangling=None):
+
+    """
+    Calls the cugraph pagerank algorithm taking in a networkX object.
+    In future releases it will maintain compatibility but will migrate more
+    of the workflow to the GPU.
+
+    Parameters
+    ----------
+    G : networkx.Graph
+
+    alpha : float, optional (default=0.85)
+        The damping factor alpha represents the probability to follow an
+        outgoing edge, standard value is 0.85.
+        Thus, 1.0-alpha is the probability to “teleport” to a random vertex.
+        Alpha should be greater than 0.0 and strictly lower than 1.0.
+
+    personalization : dictionary, optional (default=None)
+        dictionary comes from networkx is converted to a dataframe
+        containing the personalization information.
+
+    max_iter : int, optional (default=100)
+        The maximum number of iterations before an answer is returned. This can
+        be used to limit the execution time and do an early exit before the
+        solver reaches the convergence tolerance.
+        If this value is lower or equal to 0 cuGraph will use the default
+        value, which is 100.
+
+    tol : float, optional (default=1e-05)
+        Set the tolerance the approximation, this parameter should be a small
+        magnitude value.
+        The lower the tolerance the better the approximation. If this value is
+        0.0f, cuGraph will use the default value which is 1.0E-5.
+        Setting too small a tolerance can lead to non-convergence due to
+        numerical roundoff. Usually values between 0.01 and 0.00001 are
+        acceptable.
+
+    nstart : dictionary, optional (default=None)
+        dictionary containing the initial guess vertex and value for pagerank.
+        Will be converted to a Dataframe before calling the cugraph algorithm
+        nstart['vertex'] : cudf.Series
+            Subset of vertices of graph for initial guess for pagerank values
+        nstart['values'] : cudf.Series
+            Pagerank values for vertices
+
+    weight: str, optional (default=None)
+        This parameter is here for NetworkX compatibility and not
+        yet supported in this algorithm
+
+    dangling : dict, optional (default=None)
+        This parameter is here for NetworkX compatibility and ignored
+
+    Returns
+    -------
+        PageRank : dictionary
+               A dictionary of nodes with the PageRank as value
+
+    """
+    # Purposely leaving in this statement to demonstrate when cugraph
+    # algorith is being called.
+    print("Calling compat.nx pagerank")
+    local_pers = None
+    local_nstart = None
+    if (personalization is not None):
+        local_pers = cudify(personalization)
+    if (nstart is not None):
+        local_nstart = cudify(nstart)
+    return cugraph.pagerank(
+            G,
+            alpha,
+            local_pers,
+            max_iter,
+            tol,
+            local_nstart,
+            weight,
+            dangling)
```

Comment:
If the `dangling` parameter is ignored. Why are you passing it though? And this would cause an issue